packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -191,22 +187,55 @@ export interface ContainerRuntimeMessage {
     contents: any;
     type: ContainerMessageType;
 }
+export interface ISummaryConfigurationBaseSettings {
+    /* Delay before first attempt to spawn summarizing container. */
+    initialSummarizerDelayMs: number;
+
+    /**
+     * Flag that will enable changing elected summarizer client after maxOpsSinceLastSummary.
+     * This defaults to false (disabled) and must be explicitly set to true to enable.
+     */
+    summarizerClientElection: boolean;
+
+    maxAckWaitTime: number;
+    maxOpsSinceLastSummary: number;
+}
+
+export interface ISummaryConfigurationHeuristicSettings extends ISummaryConfigurationBaseSettings {
+    idleTime: number;
+    maxTime: number;
+    maxOps: number;
+    minOpsForAttemptOnClose: number;
```

Comment:
As you go through this code, please consider given better names. Things like maxOpsSinceLastSummary & minOpsForAttemptOnClose do not tell well what they are. 
Or at least add comments above each property describing the meaning and usage of it.